content/docs/user-guide/gems/reference/ros2/ros2-gem-user-guide.md

```diff
@@ -0,0 +1,209 @@
+```
+
+Note that QoS class is a simple wrapper to [`rclcpp::QoS`](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1QoS.html).
+
+### Frames
+
+`ROS2FrameComponent` is a representation of an interesting physical part of the robot. It handles spatio-temporal relationship between this part and other frames of reference. It also encapsulates namespaces, which help to distinguish between different robots and different parts of the robot, such in the case of multiple identical sensors on one robot.
+
+All Sensors and the Robot Control Component require `ROS2FrameComponent`.
+
+### Sensors
+
+Sensors are Components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
+
+- Each sensor has a configuration, including one or more Publishers.
+- Sensors publish at a given rate (frequency).
+- Some sensors can be visualized.
+
+If you intend to add your own sensor, it might be useful to look at how sensors already provided within the O3DE ROS2 Gem are implemented.
+
+Sensors can be fall into one of two categories:
+- sensors which replicate real devices to some degree of realism.
+- ground truth "sensors", which can be useful for development and machine learning.
+
+### Robot Control
+
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://index.ros.org/p/ackermann_msgs/#humble)
+  The component subscribes to these command messages on a configured topic. The topic is "cmd_vel" by default, in a namespace as dictated by ROS2Frame.
+
+To make use of received command messages, use either `AckermannControlComponent`, `RigidBodyTwistControlComponent` or `SkidSteeringControlComponent` , depending on steering type. You can also implement your own control component or use LUA scripting to handle these commands. Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`. These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present. You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/). It is possible to implement your own control mechanisms with this Component.
+
+### Vehicle Model
+
+`VehicleModelComponent` serves the purpose of converting inputs such as target velocity, steering or acceleration to physical forces on parts of a vehicle (robot). `VehicleModel` has a `VehicleConfiguration` which is used to define axles, parametrize and assign wheels. The model requires a `WheelControllerComponent` present in each wheel entity. It also uses an implementation of `DriveModel`, which converts vehicle inputs to forces acting on steering elements and wheels.
```

Comment:
This comment regards all components: 

Component documentation lives in the `/docs/user-guide/component/reference/` section. I suggest creating a task to move the component sections there. For writing component pages, please refer to the [Component Reference template](https://www.o3de.org/docs/contributing/to-docs/templates/component-reference-doc/. There's some additional information that you should considering providing about components. For example, the component's properties table should also have a `Value` and `Default` column.
For Ackermann Drive Model, refer to the formatting of "Complex set of properties" section of the Component Reference template. This splits up component properties that are organized in component groups into separate tables. This way you don't need to write long names for the components.

Reply:
I know this is a much larger request, so I'm happy to work with your team to figure out how and when we can make these adjustments to the ROS 2 Gem docs!

Reply:
Yes, that's a great idea

Reply:
Awesome thanks @chanmosq 👍